lib/active_record/virtual_attributes/virtual_fields.rb

```diff
@@ -225,12 +225,23 @@ def without_virtual_includes
 
     include(Module.new {
       # From ActiveRecord::FinderMethods
-      def find_with_associations(&block)
-        real = without_virtual_includes
-        if real.equal?(self)
-          super
-        else
-          real.find_with_associations(&block)
+      if ActiveRecord.version.to_s >= "5.2"
```

Comment:
Do we have a way to test this against 5.2 in specs?

Reply:
5.2 got a 3 new failures. these are 2 of them.

Do you want me to enable 5.2?
I can enable them, but they will fail until the 3rd failure is addressed.

Reply:
ah, yeah, can you enable them with 5.2 to demonstrate it's fixed and maybe skip the test on 5.2 for the expected failure?